src/index.ts

```diff
@@ -170,13 +170,15 @@ const graphqlServer = graphqlHTTP((req, res, params) => {
     defaultTimezone = timezone
   }
 
+  // Currently the xImpersonateUserID is only used for Braze purposes
+  const userID = xUserID || xImpersonateUserID
```

Comment:
Maybe this makes more sense here @mzikherman ?

Reply:
This is a global-ish change, but I _think_ is ok. We usually use `X-User-Id` has the user id of the token making the request. It was out of convenience - the `X-Access-Token` is enough to use `meLoader` and get the id, but we use the id in a bunch of places and so passing it in as a header (both Force/Eigen do that) lets a resolver use it directly w/o needing to fetch `meLoader` first.

That all being said...I _think_ a fallback to the impersonate header makes sense.